dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -38,9 +53,38 @@ struct eni_data_t {
     IPv4Address pl_underlay_sip;
 }
 
+typedef bit<32> RoutingType_t;
+#define ACTION_STATICENCAP      (1<<0)
+#define ACTION_TUNNEL           (1<<1)
+#define ACTION_4to6             (1<<2)
+#define ACTION_6to4             (1<<3)
+#define ACTION_NAT              (1<<4)
+#define ACTION_REVERSE_TUNNEL   (1<<5)
+#define ACTION_TUNNEL_FROM_ENCAP    (1<<6)
+
+typedef bit<32> Oid_t;
+typedef bit<8> MatchStage_t;
+#define MATCH_END           0
+#define MATCH_START         1
+#define MATCH_ROUTING0      1
+#define MATCH_ROUTING1      2
+#define MATCH_IPMAPPING0    3
+#define MATCH_IPMAPPING1    4
+#define MATCH_TCPPORTMAPPING   5
+#define MATCH_UDPPORTMAPPING   6
+
+typedef bit<16> Nexthop_t;
+
+typedef bit<8> TunnelTarget_t;
+#define TUNNEL_UNDERLAY0 1
+#define TUNNEL_UNDERLAY1 2
+
+typedef bit<8> TunnelId_t;
```

Comment:
we will need more than 8 bits for tunnel id, let's use bit<16> at least.

Reply:
Ok